README.md

```diff
@@ -66,9 +73,9 @@ The PostgreSQL modules rely on the [Psycopg2](https://www.psycopg.org/docs/) Pos
 ## Tested with ansible-core
 
 Tested with the following `ansible-core` releases:
-- 2.11
 - 2.12
 - 2.13
+- 2.14
 - current development version
 
 Ansible-core versions before 2.11.0 are not supported, in particular, ansible-base 2.10 and Ansible 2.9.
```

Comment:
```suggestion
Ansible-core versions before 2.12.0 are not supported, in particular, ansible-base 2.10 and Ansible 2.9.
```

Reply:
@hunleyd maybe we could remove what follows after 2.12.0? If you agree, feel free to update your suggestion and I'll commit it

Reply:
like that?

Reply:
exactly, thanks!